cmd/nerdctl/run.go

```diff
@@ -102,6 +109,11 @@ func setCreateFlags(cmd *cobra.Command) {
 		return []string{"always", "missing", "never"}, cobra.ShellCompDirectiveNoFileComp
 	})
 
+	// #region for init process
+	cmd.Flags().Bool("init", false, "Run an init process inside the container")
+	cmd.Flags().String("init-path", "", "Path to the nerdctl-init binary")
```

Comment:
`init-binary` would be better. The user does not need to specify the path, just like `dumb-init`.

Reply:
Make sense.

Reply:
`dockerd` has `init-path`, so `init-path` still looks good

Reply:
The `init-path` in docker has been deprecated.= =